### PR TITLE
Include addtag and deletetag test cases in DG

### PR DESCRIPTION
**Pull request by Arixeyeion**
Related to #298

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -928,29 +928,76 @@ testers are expected to do more *exploratory* testing. Alternative test cases wi
...
+#### Successful addition of one or more tags to a person
 
-   1. Prerequisites: List all persons using the `list` command. Multiple persons in the list.
+**Prerequisites:**
+1. List all persons using the `list` command. At least 5 persons are in the list.
+
+**Example Test Case**: `addtag 1 t/PRUSafe365`.
+
+**Expected Result:**
+1. `PRUSafe365` is added to the first contact on the list.
```
Comment (Javiery3889) [inline docs/DeveloperGuide.md:941]:
```suggestion
1. The tag `PRUSafe365` is added to the first contact of the list.
```

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -928,29 +928,76 @@ testers are expected to do more *exploratory* testing. Alternative test cases wi
...
+#### Successful deletion of one or more tags to a person
+
+**Prerequisites:**
+1. List all persons using the `list` command. At least 5 persons are in the list.
+2. The tags exist on the targeted person.
+
+**Example Test Case**: `deletetag 1 t/PRUSafe365`.
+
+**Expected Result:**
+1. `PRUSafe365` is deleted from the first contact.
```
Comment (Javiery3889) [inline docs/DeveloperGuide.md:977]:
```suggestion
1. The tag `PRUSafe365` is deleted from the first contact of the list.
```

**Review by Javiery3889 — COMMENTED** [review]
Just two small nits

File: docs/DeveloperGuide.md [hunk 3]
```diff
@@ -928,29 +928,76 @@ testers are expected to do more *exploratory* testing. Alternative test cases wi
...
+- `addtag 5 t/PRUActiveCash`.
 
-   1. Test case: `delete 0 t/validTag`<br>
-      Expected: No tag is deleted as 0 is not a valid index for person. Error details shown in the status message. Status bar remains the same.
+#### Invalid index, tag or command format supplied when adding one or more tags to a person
 
-   1. Test case: `delete 1 t/invalidTag`<br>
-      Expected: No tag is deleted as the tag is invalid. Error details shown in the status message. Status bar remains the same.
+**Prerequisites:**
+1. List all persons using the `list` command. Multiple persons are in the list (less than 100000).
```
Comment (KohGuanZeh) [inline docs/DeveloperGuide.md:953]: I think we should change this to `less than 100`? The cases for this should change as well.

File: docs/DeveloperGuide.md [hunk 4]
```diff
@@ -928,29 +928,76 @@ testers are expected to do more *exploratory* testing. Alternative test cases wi
...
+4. *Command Box* is cleared.
+
+**Alternative Test Cases:**
+- `deletetag 1 t/PRUSafe365 t/PRUGrowth`.
+- `deletetag 5 t/PRUActiveCash`.
+
+#### Invalid index, tag or command format supplied when deleting one or more tags to a person
+
+**Prerequisites:**
+1. List all persons using the `list` command. Multiple persons are in the list (less than 100000).
```
Comment (KohGuanZeh) [inline docs/DeveloperGuide.md:989]: I think we should change this to less than 100? The cases for this should change as well.

**Review by KohGuanZeh — APPROVED** [review]
LGTM

**Review by Javiery3889 — APPROVED** [review]
LGTM

**Pull request merged**